tajo-catalog/tajo-catalog-server/src/main/java/org/apache/tajo/catalog/dictionary/InfoSchemaMetadataDictionary.java

```diff
@@ -91,7 +91,7 @@ public String getSystemDatabaseName() {
     return systemTableNames;
   }
   
-  private TableDescriptor getTableDescriptor(String tableName) {
+  private TableDescriptor getTableDescriptor(String tableName) throws UndefinedTableException {
```

Comment:
According to my design, it should throw an exception instead of NULL. I fixed. Thanks!